spec/orders_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'orders'
+# require 'sms'
+
+describe Orders do
+
+  let(:menu) { double :menu, :menu_items => { :pie => 4.5, :chips => 2.50 } }
+  let(:mockbill) { double :mockbill, :bill => 14 }
+  let(:bill_klass) { double :bill_klass, :new => mockbill }
+
+  it 'chooses the menu' do
+    expect(subject.choose_menu(menu)).to eq menu.menu_items
+  end
+
+  it 'responds to 2 arguments' do
+    expect(subject).to respond_to(:cust_order).with(2).arguments
+  end
+
+  it "can place multiple orders" do
+    subject.choose_menu
```

Comment:
you haven't made use of your mock menu object here, so the test is not isolated - it relies on a real instance of the Menu class. Should be a quick and easy fix!